tests/integration_tests/supremm_setup_expect.py

```diff
@@ -55,9 +69,9 @@ def main():
         p.expect("DB port")
         p.sendline()
         p.expect("DB Admin Username")
-        p.sendline()
+        p.sendline("xdmod")
         p.expect("DB Admin Password")
-        p.sendline()
+        p.sendline("xdmod123")
```

Comment:
Please explain why this changed

Reply:
This change is here from the larger set of changes I was making to the tests that include an external Mariadb service (instead of included in the job_performance 10.0 image). This was the password for the service. Because that feature for the tests was reverted, I can just remove for now and add it back when the full testing changes are implemented.

Reply:
Resolved in https://github.com/ubccr/supremm/pull/259/commits/dcfe32158a538fbb3ec8cd10ad18dd3cea87835a